src/darwin/Framework/CHIPTests/MTRXPCListenerSampleTests.m

```diff
@@ -439,6 +439,19 @@ - (void)controller:(MTRDeviceController *)controller commissioningComplete:(NSEr
     _expectation = nil;
 }
 
+- (void)controller:(MTRDeviceController *)controller commissioningComplete:(NSError *)error deviceId:(NSNumber * _Nullable)deviceId
+{
+    XCTAssertEqual(error.code, 0);
+    [_expectation fulfill];
+    _expectation = nil;
+}
+
+- (void)controller:(MTRDeviceController *)controller readCommissioningInfo:(NSDictionary<NSString *, id> * _Nullable)info
+{
+    [_expectation fulfill];
```

Comment:
This also seems like strange thing to do.  Won't this fulfill the "commissioning is done" expectation even though commissioning is not in fact done?

Reply:
remove it.